types/steamapi/index.d.ts

```diff
@@ -89,36 +90,37 @@ declare class SteamAPI {
      * @param host Host to request
      * @returns Server info
      */
-    getServers(host: string): Promise<Server[]>;
+    getServers(host: string): Promise<SteamAPI.Server[]>;
 
     /**
      * Get users achievements for app id.
      * @param id User ID
      * @param app App ID
      * @returns Achievements
      */
-    getUserAchievements(id: string, app: string): Promise<PlayerAchievements>;
+    getUserAchievements(id: string, app: string): Promise<SteamAPI.PlayerAchievements>;
 
     /**
      * Get users badges.
      * @param id User ID
      * @returns Badges
      */
-    getUserBadges(id: string): Promise<PlayerBadges>;
+    getUserBadges(id: string): Promise<SteamAPI.PlayerBadges>;
 
     /**
      * Get users bans.
      * @param id User ID(s)
      * @returns Ban info
      */
-    getUserBans(id: string): Promise<PlayerBans | PlayerBans[]>;
+    getUserBans(id: string[]): Promise<SteamAPI.PlayerBans[]>;
+    getUserBans(id: string): Promise<SteamAPI.PlayerBans>;
```

Comment:
```suggestion
    /**
     * Get users bans.
     * @param id User ID
     * @returns Ban info
     */
    getUserBans(id: string): Promise<SteamAPI.PlayerBans>;
```

Reply:
JSDoc automatically propagates over multiple overloads, so it shouldn't be necessary.

VSCode:
![image](https://user-images.githubusercontent.com/6274312/140763985-342fce0f-97b0-4004-8777-f6a22734dd0c.png)

Reply:
Didn't work for me 😔:
![](https://media.discordapp.net/attachments/707505654218358818/907283615191285800/jsdoc.gif)

Reply:
Must be IDE specific. I'll add the changes then. 😄